sensor/admission-control/manager/evaluate_runtime.go

```diff
@@ -121,7 +121,7 @@ func (m *manager) evaluatePodEvent(s *state, req *admission.AdmissionRequest, ev
 
 		alerts, err := m.kickOffImgScansAndDetect(fetchImgCtx, s, getAlertsFunc, deployment)
 		if err != nil {
-			return nil, false, err
+			return nil, false, errors.Wrap(err, "runtime detection for deployment and event failed")
```

Comment:
```suggestion
			return nil, false, errors.Wrap(err, "runtime detection for deployment and event")
```

and this pattern can be applied to all changes in this file, so I will stop commenting on this type of change.